test/test_ecs_monitor.py

```diff
@@ -574,6 +649,75 @@ def test_monitor_image_does_not_match(
 
         self.assertRaises(ImageDoesNotMatchError, lambda: [e for e in events])
 
+    def test_get_ecs_service_events(self):
```

Comment:
We're testing this method via the public interface in `test_monitor_exited_when_the_deployment_is_stable` so we can probably just remove this test altogether.